package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@iobroker/adapter-dev",
-  "version": "1.3.0",
+  "version": "1.3.1",
```

Comment:
Please do not increase version. The release script will do this

Reply:
reverted